lib/perl/Genome/Command/DelegatesToResult.pm

```diff
@@ -15,13 +15,21 @@ class Genome::Command::DelegatesToResult {
         },
     ],
     has_optional_input => [
+        sponsor => {
+            is => 'UR::Object',
+            doc => 'The responsible party for this result (defaults to the current user)',
+        },
+        requestor => {
```

Comment:
Why not make `requestor` required instead of manually implementing a similar check in `__errors__`?